pkg/daemon/daemon.go

```diff
@@ -492,6 +504,27 @@ func (d *UnikDaemon) initialize() {
 				mountPoints = strings.Split(mntStr, ",")
 			}
 
+			sizeStr := req.FormValue("size")
```

Comment:
Currently, the size is only used by OSv Qemu and OSv OpenStack provider. Would it make sense to check this and report a warning of `--size` is used in any other case? So, `size` would only be read in these two cases, otherwise a warning would be printed informing the user that the `size` has been ignored.